### PR TITLE
Add CloudCanal loading related docs

### PR DESCRIPTION
**Pull request by KaimingWan**
Add CloudCanal loading related docs

### CODE REVIEW

**Comment by wangsimo0** [comment]
There are some issues in markdownlint that need to be fixed.

**Comment by KaimingWan** [comment]
@wangsimo0 Two markdown files are reformated with the hint from markdown lint. Please check again, thanks~

**Comment by KaimingWan** [comment]
Two markdown files are reformated with the hint from markdown lint. Please check again, thanks. @wangsimo0

File: loading/CloudCanal-loading.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+### 添加数据源
+
+- 登录 CloudCanal 平台
+- 数据源管理->新增数据源
+- 选择自建数据库中StarRocks
+
+![image.png](../assets/cloudcanal/load_2.png)
+
+> Tips:
+> - Clinet地址: 为StarRocks提供给MySQL Client的服务端口，CloudCanal主要用其查询库表的元数据信息
```
[the remark this answers is not in the document]
Comment (KaimingWan) [reply loading/CloudCanal-loading.md]: @wangruin Markdown files are edited again, please check~

File: loading/CloudCanal-loading.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## 下载安装
+
+[CloudCanal最新版下载地址](https://cloudcanal-community.oss-cn-hangzhou.aliyuncs.com/latest/cloudcanal.7z)
+
+[CloudCanal社区版安装部署参考文档](https://doc-cloudcanal.clougence.com/operation/install_linux)
+
+## 功能说明
+
+- StarRocks对端版本支持：支持StarRocks版本为1.18x、1.19x和2.0.x
```
Comment (wangruin) [inline loading/CloudCanal-loading.md:16]: Better use 1.18.x、1.19.x ?

File: loading/CloudCanal-loading.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+- 数据源管理->新增数据源
+- 选择自建数据库中StarRocks
+
+![image.png](../assets/cloudcanal/load_2.png)
+
+> Tips:
+>
+> - Client地址: 为StarRocks提供给MySQL Client的服务端口，CloudCanal主要用其查询库表的元数据信息
+>
+> - Http地址： Http地址主要用于接收CloudCanal数据导入的请求
```
Comment (wangruin) [inline loading/CloudCanal-loading.md:45]: Please use Chinese colon here.

**Review by wangsimo0 — APPROVED** [review]

**Pull request merged**